docs/Contributing/product-groups/README.md

```diff
@@ -0,0 +1,44 @@
+# Fleet Product Groups
+
+Fleet is organized into three main product groups:
+
+## MDM
+
+The [MDM product group](mdm/README.md) focuses on Mobile Device Management, including:
+
+- Apple MDM (macOS, iOS)
+- Windows MDM
+- Android MDM
+- End user authentication
+- Disk encryption
+- Automated device enrollment
+
+## Orchestration
+
+The [Orchestration product group](orchestration/README.md) focuses on device orchestration using osquery, including:
+
+- Live queries
+- Scheduled queries
+- Query packs
```

Comment:
Do we want to even list this anymore? Same thing with file carving; we haven't touched those features in awhile so they're just noise when listed out.

We're also missing policies and scripts here.

Reply:
Yes, I think we need to list it because it's still supported and some customers still use it.